tests/test_clinical_trials_data.py

```diff
@@ -0,0 +1,25 @@
+
+from clashboard.clinical_trials_data import ClinicalTrialsData
+from tests.mocks.mock_db import MockDB
+
+
+def test_new_instance_has_no_group():
+
+    db = MockDB(None)
```

Comment:
The MockDB goes away.  My comments above assume this class is being re-purposed to be the Mock version of the interface.